### ISSUE TITLE
Why previous universal languages failed?

### ISSUE THREAD

**Issue by rottenoats** [TerraVera/TerraVeraLang#1]
An interesting subject of discussion is to see how previous attempts at universal languages failed and how this one can succeed.

**Comment by Zyfast** [comment]
All the previous attempts to create the universal language failed, because people tend to modify languages depending on different factors, for example, geographical position, cultural aspect, even weather. That is why Esperanto learnt in China differs from Esperanto learnt in Chicago, for example. BUT our idea is not to create one and only unchangeable language. We want to develop kind of a linguistic platform which will be structured in such a way that people will behave humanly, since languages influence our way of thinking and our personality. Once created, this unifying language will develop naturally and bring into life other languages.

**Comment by rottenoats** [comment]
@BouncingRabbit I think that's a good point:

> because people tend to modify languages depending on different factors

Unless there is a spoken language out there that has specific versioning, language documentation has never ever been kept. Language advancement kind of branches off differently depending on location.

I think a good thing about this idea, is that it uses modern technology to keep track of advancement and changes within the language.

I also believe that a language should be allowed to branch out into different language projects, if a group decided to do so. Similar to the different Linux distributions that exist. A fork of the original project, but with a different name (since a fork of project, is not the original project).

Much like the distinction between American English and British English. You could say Chinese Esperanto and American Esperanto are a fork (computing term for copy) of the original language.

I'd still like to see some other universal language examples. It would help our case.

**Comment by Zyfast** [comment]
> _Unless there is a spoken language out there that has specific versioning, language documentation has never ever been kept. Language advancement kind of branches off differently depending on location_ 

Yes, and there's also a psychological aspect, meaning that people identify themselves through languages. That is why the idea to create a well structured unifying language with the purpose to positively change the way the society thinks is a very decent one. 

Do you have an idea how to track the changes of this new language in the future btw? 

> _I'd still like to see some other universal language examples. It would help our case._

Well, Esperanto is the only one that actually had a relative success, the reason of which is that its creators strongly believe in globalization. I've never heard about projects similar to ours though. But yeah, let's search for other unifying languages.

**Comment by rottenoats** [comment]
> Do you have an idea how to track the changes of this new language in the future btw?

Yes, of course. That's the whole point of using github that is based on the git technology. Here is a quick definition:
> Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency.

source: https://git-scm.com/

Regarding Universal languages, here are some interesting articles:

Talks about some issues Esperanto faced when talking about "communication" between the different R&D teams from around the world.. which would easily be solved in using github for example.
https://www.cebglobal.com/blogs/global-rd-alternatives-to-esperanto/

This article talks about a "top 10 universal languages", it's obviously not real universal languages but something important to take into account while developing TerraVera.
https://www.thetoptens.com/universal-languages/

This article talks about the "characteristica universalis" which a german polymathic genious that came up with a pictographic universal language.
https://en.wikipedia.org/wiki/Characteristica_universalis

Ido is a reformed version of Esperanto that seems to have similar goals as making sure it's a "regular" language.
https://en.wikipedia.org/wiki/Ido_language

Interlingua is also another project started in the 1930s, inspired from different languages apparantely.
https://en.wikipedia.org/wiki/Interlingua

aUI is a universal language developed in the 1950s. 
https://en.wikipedia.org/wiki/AUI_(constructed_language)

This is also an interesting find regarding the "Global language system", I think it's worth reading.
https://en.wikipedia.org/wiki/Global_language_system

**Comment by rottenoats** [comment]
The conclusion I see is that, Esperanto and other universal languages do not see their desired "success" is due to a variety of reasons that can attributed to the findings of some of the articles I previously linked.

A universal language should be
- Linked to a clear message
- Should incorporate Music and Rythm
- Should incorporate a form of Images
- Should follow a set of basic rules, that together can create complex strucutres
- Should be unique and not easily associated with any specific existing language or system (such as Esperanto which resembles Spanish and Latin for example)
- Should have it's own counting system

Those were some ideas off the top of my head, I'll edit it when I have more I suppose.

**Comment by Zyfast** [comment]
Oooops sorry I didn't get it at the beginning, thanks for the explanation. 

You're a wikipedia warrior haha Ok, I'll read it. You too, please, do read Korzybski, because we need to know how languages influence human perception. Since we want our language to positively structure the new society, we should read a lot of psycholinguistic stuff. 

As for the universal languages in general, I remember, in every uni I studied, they kept saying that it wouldn't succeed because any language needs development. It never convinced me ;) To have a language that would be used as one and only way of communication by every single person in the world without any modification is, indeed, a pure utopia. While our goal is completely different!

**Comment by Zyfast** [comment]
> Should be unique and not easily associated with any specific existing language or system

Definitely! What can also be done is the analysis of the existing groups of languages. We could chose the most useful characteristics and functions and apply them to the new language. 

And yeah, the language should also be logical.

**Comment by rottenoats** [comment]
> You too, please, do read Korzybski

@BouncingRabbit already read half of it 💃 

> As for the universal languages in general, I remember, in every uni I studied, they kept saying that it wouldn't succeed because any language needs development. It never convinced me ;) To have a language that would be used as one and only way of communication by every single person in the world without any modification is, indeed, a pure utopia. While our goal is completely different!

That's the idea behind this language, a community-driven effort. Constant development and releases of minor and major versions.

**Comment by Zyfast** [comment]
> already read half of it 

YES! Well done! It is my bible in a sense haha

**Comment by Zyfast** [comment]
> That's the idea behind this language, a community-driven effort. Constant development and releases of minor and major versions

Which makes a difference!

**Comment by Zyfast** [comment]
Hey! Watch this video 
https://www.youtube.com/watch?v=q0whherUIeo
The guy is criticizing the idea of creating a universal language, but it's not the point, we don't care. I thought it would be interesting to find out how the Loglan/Lojban language works. It's pretty much the idea that you had, with some drawbacks that the guys explains well

**Comment by rottenoats** [comment]
@Zyfast that is a good video thanks for sharing. It comes down to the same issue as ever, that languages eventually branch out because the inventor doesn't want to accept certain changes and the branches become community driven languages.

Loglan is interesting but it resembles spanish a bit too much in my opinion, just like esperanto.

The goal would be to make community driven from the start, thus to avoid the issues the original creators encountered. A living language should accept and review any contributions.

**Comment by Zyfast** [comment]
> The goal would be to make community driven from the start, thus to avoid the issues the original creators encountered. A living language should accept and review any contributions.

Yeah, that's what we already discussed. And the video showed well what previous creators didn't get. Language should live, i.e. develop itself = change itself. It's absolutely normal. Otherwise, it would stop the society from development. But the basis should be created, the one that is simple, logical, and human. Which is again absolutely natural, since all the languages we speak came from one unique mother language.
